src/vivilux/layers.py

```diff
@@ -174,6 +176,24 @@ def RunProcesses(self):
     def AddProcess(self, process: Process):
         process.AttachLayer(self)
 
+    def AddMonitor(self, monitor: Monitor):
+        self.monitors.append(monitor)
+
+    def UpdateMonitors(self):
```

Comment:
If the monitor is added but turned off, it should not be updated. Should there be a boolean check here?

Reply:
I will add a `enable` boolean variable to the `Monitor` class in order to update the monitor only when `enable=True`.